.gitignore

```diff
@@ -2,6 +2,7 @@
 
 # Dependencies
 node_modules/
+package-lock.json
```

Comment:
This needs to be tracked in git, since it allows us to pin the specific version that we use. As a side effect of running `npm install`, some times it gets modified. If you want to achieve the same effect without updating this file, use `npm ci` (that is, for installing the environment, not for installing a new package)

Reply:
should I remove this from the changes then?

Reply:
Yes, no `package-lock.json` in `.gitignore`